### ISSUE TITLE
How can I access output of a layer in my model in a custom loss function?

### ISSUE THREAD

**Issue by BitFloyd** [fchollet/keras#5466]
def objective_custom(y_pred,layer_index,model):
    get_op = theano.function ([model.layers[0].input],              model.layers[0].call(model.layers[0].input),allow_input_downcast=True)
    y_true = get_op(model.layers[0].input)
    return K.mean(K.square(y_pred - y_true), axis=-1)

How can I access the output of a layer from my loss function?
